### PR TITLE
docs: update faq

### PR DESCRIPTION
**Pull request by bht991002**
review needed

### CODE REVIEW

File: docs/faq-and-others/faq.md [hunk 1]
```diff
@@ -70,3 +70,11 @@ You can use the [`COPY TO` command](https://docs.greptime.com/reference/sql/copy
 ## TSDB features that you concern
 
 Please refer to [features that you concern](/user-guide/concepts/features-that-you-concern.md).
+
+## Does GreptimeDB support auto-increment primary keys similar to MySQL?
+
+Not yet.
```
Comment (nicecui) [inline docs/faq-and-others/faq.md:76]: Will it be supported in the future?

Comment (bht991002) [reply docs/faq-and-others/faq.md:76]: Jiachun says now we don't have the plan to support it since time-series scenario is not common

Comment (nicecui) [reply docs/faq-and-others/faq.md:76]: a little advice: explain further why a time-series database does not require auto-increment, in order to help users better understand time-series usage scenarios.

**Review by nicecui — APPROVED** [review]

File: docs/faq-and-others/faq.md (hunk 1 above)
Comment (fengjiachun) [reply docs/faq-and-others/faq.md]: I suggest deleting this question as I believe that auto-increment ID is more appropriate for TP database scenarios and not commonly used in time series.

**Review by xtang — APPROVED** [review]

**Pull request merged**